Makefile

```diff
@@ -136,7 +135,7 @@ test-short:
 ## test-e2e: Run end to end tests via knuu. This command requires a kube/config file to configure kubernetes.
 test-e2e:
 	@echo "--> Running end to end tests"
-	@KNUU_NAMESPACE=test KNUU_TIMEOUT=20m E2E_LATEST_VERSION=$(shell git rev-parse --short main) E2E_VERSIONS="$(ALL_VERSIONS)" go test ./test/e2e/... -timeout 20m -v
+	go run ./test/e2e $(filter-out $@,$(MAKECMDGOALS))
```

Comment:
🆒 I think this enables developers to pass extra arguments to `make test-e2e` and they get plumbed in to `go run`...

Reply:
yes, yes